bang/utils.py

```diff
@@ -65,6 +66,15 @@ def rarity_to_stars_images(rarity):
         un='' if rarity in models.Card.TRAINABLE_RARITIES else 'un',
     ) * rarity
 
+def generateDifficulty(difficulty, space):
+    note_image = staticImageURL('note.png')
+    val = mark_safe(u'{big_images}{small_images}'.format(
+    big_images=(u'<img src="{}" class="song-big-note">'.format(note_image) * (difficulty // 5)),
+    small_images=(u'<img src="{}" class="song-small-note">'.format(note_image) * (difficulty % 5)),))
+    if space is True:
+        val += '<br />'
+    return val
```

Comment:
I would move the `mark_safe` to where you call the function (it's an unsafe function so it's better to not have it hidden) and add the `<br>` where you need it after the call:

```
mark_safe(u'{}<br>'.format(generateDifficulty(diff)))
```

Reply:
I try to avoid using `+` for strings because with `u''.format()` it's easier to make sure all the string are unicode strings by just adding a `u` before. For small examples like that it's fine though. Just make sure the string also starts with `u`.